From 5825d466afa6e463778bfa43cb5a62f1bcb146af Mon Sep 17 00:00:00 2001
From: Jiangli Yuan <jlyuan@motorola.com>
Date: Tue, 17 May 2016 13:43:55 -0700
Subject: [PATCH 892/959] IKSWM-38800: 8952: Reduce the APC voltage bump to
 25mV

 Reduce the vdd_apc voltage bump to 25mV for corner 4,5

Change-Id: I1c98ea9775964f725264d75c328b08d8a19aca6a
Signed-off-by: Jiangli Yuan <jlyuan@motorola.com>
Reviewed-on: http://gerrit.mot.com/857653
SLTApproved: Slta Waiver <sltawvr@motorola.com>
SME-Granted: SME Approvals Granted
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/boot/dts/qcom/msm8952-regulator.dtsi | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm/boot/dts/qcom/msm8952-regulator.dtsi b/arch/arm/boot/dts/qcom/msm8952-regulator.dtsi
index 47fce053f8e..75a3e228995 100644
--- a/arch/arm/boot/dts/qcom/msm8952-regulator.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8952-regulator.dtsi
@@ -440,19 +440,19 @@
 			<0    16      33>, /* NOM + 10mv, TURBO + 20mv */
 			<0    0        0>;
 		cpr-virtual-corner-init-voltage-adjustment =
-			<0 0 0 50000 50000 0 0 0>,
-			<0 0 0 50000 50000 0 0 0>,
-			<0 0 0 50000 50000 0 0 0>,
-			<0 0 0 50000 50000 0 0 0>,
-			<0 0 0 50000 50000 0 0 0>,
-			<0 0 0 50000 50000 0 0 0>;
+			<0 0 0 25000 25000 0 0 0>,
+			<0 0 0 25000 25000 0 0 0>,
+			<0 0 0 25000 25000 0 0 0>,
+			<0 0 0 25000 25000 0 0 0>,
+			<0 0 0 25000 25000 0 0 0>,
+			<0 0 0 25000 25000 0 0 0>;
 		qcom,cpr-virtual-corner-quotient-adjustment =
-			<0 0 0 82 82 0 0 0>,
-			<0 0 0 82 82 0 0 0>,
-			<0 0 0 82 82 0 0 0>,
-			<0 0 0 82 82 0 0 0>,
-			<0 0 0 82 82 0 0 0>,
-			<0 0 0 82 82 0 0 0>;
+			<0 0 0 41 41 0 0 0>,
+			<0 0 0 41 41 0 0 0>,
+			<0 0 0 41 41 0 0 0>,
+			<0 0 0 41 41 0 0 0>,
+			<0 0 0 41 41 0 0 0>,
+			<0 0 0 41 41 0 0 0>;
 		qcom,cpr-floor-to-ceiling-max-range =
 			<(-1) 30000 30000 30000 30000 50000 50000 50000>,
 			<(-1) 30000 30000 30000 30000 50000 50000 50000>,
-- 
2.11.0

